test/e2e/opm_test.go

```diff
@@ -428,10 +409,20 @@ var _ = Describe("opm", func() {
 	}
 
 	Context("using docker", func() {
-		IncludeSharedSpecs("docker")
+		if err := exec.Command("docker").Run(); err == nil {
+			IncludeSharedSpecs("docker")
+		} else {
+			GinkgoT().Logf("container tool docker not found - skipping docker-based opm e2e tests: %s", err)
+			return
+		}
```

Comment:
Just a quick drive-by, but you could deflate this to a single conditional by reversing this logic and exiting early.